presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -504,7 +504,7 @@ private MaterializedResult executeInternal(Session session, @Language("SQL") Str
             TaskContext taskContext = createTaskContext(executor, session);
 
             List<Driver> drivers = createDrivers(session, sql, outputFactory, taskContext);
-            drivers.stream().map(closer::register);
+            drivers.forEach(closer::register);
```

Comment:
yes, it was warning me, but somehow when I was writing this `.map(...)` I have missed that :(